mysystehis/main.tex

```diff
@@ -0,0 +1,1155 @@
+ 	   \begin{figure}[h]
+ 	   	 \centering
+ 	   	 \begin{minipage}{1\textwidth}
+ 	   	 	\centering
+ 	   	 	\includegraphics[width=0.75\linewidth]{intructions/3.png}
+ 	   	 	\captionof{figure}{Sections}
+ 	   	 	\label{fig:test4}
+ 	   	 \end{minipage}
+ 		 \end{figure}  
+ 		 
+ 	Besides, in order to make the terrain looks better, the user can also raise or lower the terrain to create mountains, river. After i finish creating the terrain, all i have to do is drag and drop other objects/prefabs into the scene, put the textures corresponding to each object, while these prefabs/objects can be found on unity assets store or from free3d website as i mentioned from the introduction. 
+ 	 \paragraph{Player}
+ 	 To be able to interact with the objects, NPCs, a player should be added into the scene. For example, i use this little robot for my player, this model is free to download, use: \href{https://free3d.com/3d-model/bb8-35865.html}{https://free3d.com/3d-model/bb8-35865.html}.
+ 	 
+ 	 \begin{figure}[h]
+ 	 	\centering
+ 	 	\begin{minipage}{1\textwidth}
+ 	 		\centering
+ 	 		\includegraphics[width=0.75\linewidth]{intructions/4.png}
+ 	 		\captionof{figure}{Character example}
+ 	 		\label{fig:test5}
+ 	 	\end{minipage}
+ 	 \end{figure}
+ 	  
+ 	 Initially, the player will not move without any scripts attached to it, so i write a simple C\# script to control the player using visual studio and then attach the script to the player. Although there's a character controller prefab in the Unity's standard asset, which can be used, controlled instantly without doing anything. In this project, i'll not use character controller prefab, so that i can easily customize my character controller script later. \\[0.25cm] By default, when initializing a unity project, there's already a camera called "Main camera", which is what the user can see from the game view in the hierarchy. This camera will not follow my character as i move, but i'll explain how to make the camera follow my character later.  
+ 	 \\[0.25cm] From now, i'm going to explain how can i implement character controller to move my character. Firstly, i have to find the input by go to Edit -> Project Settings -> Input. In the input section, as you can see, the Unity already implement 2 inputs named : "Horizontal" , "Vertical", along with Negative Button and Positive Button, and i can also add my own custom input in project setting. This is where i can move my character left and right, forward and backward. After finding the input, i can get the input by using: 
```

Comment:
Huy sua lai tat ca nhung cho dung "the Unity" thanh "Unity" nhe. Unity ko co "the" vi no la danh tu xac dinh.